samples/cf-sample-application/README.md

```diff
@@ -45,6 +45,16 @@ cd cloud-sdk-js
 git sparse-checkout set samples/cf-sample-application
 ```
 
+### Create oData Client
+
+Due to intellectual property reasons we are not allowed to include SAP service definitions in this open source repository.
```

Comment:
This phrasing seems a bit odd to me.

Maybe something like "For legal reasons, this repository can't contain the SAP service definitions."